### PR TITLE
Feature database

### PR DESCRIPTION
**Pull request by nikolagudelj**

### CODE REVIEW

**Comment by nikolagudelj** [comment]
[[E0.2] Replace Json storage with SQL [8]](https://trello.com/c/hPZ8rEAP/13-e02-replace-json-storage-with-sql-8)

File: HealthcareBase/Model/Database/MySqlContext.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace HealthcareBase.Model.Database
+{
+    class MySqlContext : DbContext
+    {
+        // Database access password
+        private static string password = "chc6kcxhd";
```
Comment (FmasterofU) [inline HealthcareBase/Model/Database/MySqlContext.cs:27]: Hell no.

**Comment by FmasterofU** [comment]
[PR Feature Database](https://trello.com/c/wHj4Yhoi/49-pr-feature-database)

**Review by FmasterofU — CHANGES_REQUESTED** [review]
Secrets must not be present in source code!
Your options are to use Environment Variables, Secret Manager(better), Azure Key Vault(I'm not sure how this works, but that could be the best option) ...
Make sure to use a different DB password after this and a sincere advice is to never use this password again, anywhere, since it's been posted online.
Please ask, and do expect help from @PSW-2020-ORG9/devops team in resolving this issue.

**Review by FmasterofU — APPROVED** [review]
Secret (db conn) issue delegated to ASP.NET(+base dll) project.

**Review by hi-imAndy — APPROVED** [review]

**Review by rammba — APPROVED** [review]

**Comment by hi-imAndy** [comment]
[PR Feature database](https://trello.com/c/3TQYSdRf/58-pr-feature-database)

**Pull request merged**